Revert part of http://cgit.freedesktop.org/fontconfig/commit/fontconfig.pc.in?id=338ffe6b886ad4ba86ff471cb59c4a5e5ffbe408
We really dont do static linking so its not going to cause issues for us but as such this is harming us by adding absolute
paths into -I options into fontconfig.pc which is them prepended with sysroot when read back by cross pkg-config and
we end up with -I <sysroot>/<sysroot>/usr/include/fontconfig in CFLAGS of clients which depend on it. one such example
is matchbox-panel-2 and it was working fine with gcc < 5 because it tolerated non-existing paths in -I flags but gcc-5
gently refuses and rightly so.

Upstream-Status: Inappropriate [OE-Specific]
Signed-off-by: Khem Raj <raj.khem@gmail.com>

diff --git a/fontconfig.pc.in b/fontconfig.pc.in
index 61b35fb..5bc9bab 100644
--- a/fontconfig.pc.in
+++ b/fontconfig.pc.in
@@ -15,4 +15,4 @@ Requires: @PKGCONFIG_REQUIRES@
 Requires.private: @PKGCONFIG_REQUIRES_PRIVATELY@
 Libs: -L${libdir} -lfontconfig
 Libs.private: @ICONV_LIBS@ @PKG_EXPAT_LIBS@
-Cflags: -I${includedir} @ICONV_CFLAGS@ @PKG_EXPAT_CFLAGS@
+Cflags: -I${includedir}
